index.html

```diff
@@ -0,0 +1,130 @@
+                    <li class="program-item">
+                        <img src="./images/program/Lecture_icon.svg" alt="lecture-icon">
+                        <h4>Lecture</h4>
+                        <p>Listen to the instructors from varioud countries about the message of sharing and openeing.</p>
+                    </li>
+                    <li class="program-item">
+                        <img src="./images/program/Exhibition_icon.svg" alt="lecture-icon">
+                        <h4>Content</h4>
+                        <p>Top content at camp for backend devs: real-world and market relevant for the backend
+                            developers.</p>
+                    </li>
+                    <li class="program-item">
+                        <img src="./images/program/forum_icon.svg" alt="lecture-icon">
+                        <h4>Contacts</h4>
+                        <p>Connect with diverse devs, gain global insights and expertise. Learn from their prev Experiences.</p>
+                    </li>
+                    <li class="program-item">
+                        <img src="./images/program/Workshop_icon.svg" alt="lecture-icon">
+                        <h4>Workshops</h4>
+                        <p>Real use-cases for backend learning: practice and gain insight. see the real use-cases.</p>
+                    </li>
+                    <li class="program-item">
+                        <img src="./images/program/Ignite_icon.svg" alt="lecture-icon">
+                        <h4>Network</h4>
+                        <p>get opportunities to network with Nodejs Camp affiliates around the world,  after the summit.</p>
+                    </li>
+                </ul>
+                <a href="#" class="join-btn" id="join-btn">Join CodeCamp 2023 </a>
+            </div>
+        </section>
+        <section class="instructor" id="instructor">
+            <div class="container">
+                <h3 class="instructor-title">Camp's Instructors</h3>
+                <ul class="instructor-main"></ul>
+                <button type="button" class="more"> MORE <img src="./images/instructor/more_icon.svg"></button>
+            </div>
+        </section>
+        <section class="home-partner" id="partner">
+            <div class="container">
+                <h3 class="home-partner-title">Partner</h3>
+                <ul class="home-partner-main">
+                    <li class="home-partner-item"><img src="./images/partners/Microverse.png" alt="Microverse logo">
+                    </li>
+                    <li class="home-partner-item"><img src="./images/partners/microsoft.png" alt="Microsoft logo"></li>
+                    <li class="home-partner-item"><img src="./images/partners/IBM.png" alt="IBM logo"></li>
+                    <li class="home-partner-item"><img src="./images/partners/Coursera.png" alt="Courser logo"></li>
+                    <li class="home-partner-item"><img src="./images/partners/codeCamp.png"></li>
```

Comment:
- [ ] Please 🙏🏽 use the `alt` attribute for the image for the best practices of HTML & CSS.